nixos/modules/virtualisation/waagent.nix

```diff
@@ -192,6 +192,15 @@ let
             If set to `null`, the system defaults are used.
           '';
         };
+
+        OpensslPath = lib.mkOption {
```

Comment:
Can we patch the waagent source to pick up `openssl` from the $PATH we already provide it with, as written in https://github.com/NixOS/nixpkgs/pull/362101#issuecomment-2602436504?

This feels cleaner than adding a NixOS option here.

Reply:
Done as suggested, thank you!